<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="1" total="2" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="RegressionSuite" duration-ms="31" started-at="2018-04-10T15:23:20Z" finished-at="2018-04-10T15:23:21Z">
    <groups>
      <group name="Regression">
        <method signature="Grouping.findBrokenImages()[pri:0, instance:testCases.Grouping@d8355a8]" name="findBrokenImages" class="testCases.Grouping"/>
      </group> <!-- Regression -->
    </groups>
    <test name="RegressionTestCases" duration-ms="31" started-at="2018-04-10T15:23:20Z" finished-at="2018-04-10T15:23:21Z">
      <class name="testCases.Grouping">
        <test-method status="FAIL" signature="findBrokenImages()[pri:0, instance:testCases.Grouping@d8355a8]" name="findBrokenImages" duration-ms="16" started-at="2018-04-10T20:53:20Z" finished-at="2018-04-10T20:53:21Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at testCases.Grouping.findBrokenImages(Grouping.java:29)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- findBrokenImages -->
      </class> <!-- testCases.Grouping -->
    </test> <!-- RegressionTestCases -->
  </suite> <!-- RegressionSuite -->
</testng-results>
